src/components/eip3643/Admin.tsx

```diff
@@ -50,20 +50,23 @@ export default function Admin() {
     setTokenSelected(tokenItem || null);
   };
 
-  const tokensData = ownTokens.map((token) => ({
-    value: token.address,
-    label: `${token.name} ${token.address}`,
-  }));
-
   return (
     <>
       {!isDeploy && (
         <Stack spacing={4} align="center">
           <MenuSelect
-            buttonProps={{ style: { width: "50%" } }}
+            buttonProps={{ style: { width: "55%" } }}
```

Comment:
I have some concerns about flexibility of approach of combining the static width and cutting the length of content text. Considering potential amount of different screen sizes and mobile devices.
Would something like `ellipsis` css help us to prevent overflowing text on the dropdown ^ v symbols?

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow